modules/check_in/app/sidekiq/check_in/travel_claim_base_job.rb

```diff
@@ -91,27 +114,8 @@ def va_notify_send_sms(opts)
       notify_client.send_sms(phone_number:, template_id:, sms_sender_id:, personalisation:)
     end
 
-    def log_send_sms_failure(retry_attempt)
-      logger.info({ message: "Sending SMS failed, attempt #{retry_attempt} of #{MAX_RETRIES}" })
-    end
-
-    def handle_error(ex, opts = {})
-      template_id = opts[:template_id]
-      log_exception_to_sentry(
-        ex,
-        { phone_number: opts[:mobile_phone].delete('^0-9').last(4), template_id:,
-          claim_number: opts[:claim_number] },
-        { error: :check_in_va_notify_job, team: 'check-in' }
-      )
-      if FAILED_CLAIM_TEMPLATE_IDS.include?(template_id)
-        tags = if 'oh'.casecmp?(opts[:facility_type])
-                 Constants::STATSD_OH_SILENT_FAILURE_TAGS
-               else
-                 Constants::STATSD_CIE_SILENT_FAILURE_TAGS
-               end
-        StatsD.increment(Constants::STATSD_NOTIFY_SILENT_FAILURE, tags:)
-      end
-      StatsD.increment(Constants::STATSD_NOTIFY_ERROR)
+    def log_send_sms_failure(attempt_number)
+      logger.info({ message: "Sending SMS failed, attempt #{attempt_number} of #{MAX_RETRIES}" })
```

Comment:
I think this should this be `MAX_RETRIES + 1` since the first attempt is technically not a retry and this would eventually result in the last message being `attempt 13 of 12`? I

Reply:
@wullaski yes makes sense, if you check line 54 that is taken care of before the number is passed into the method

Reply:
@wullaski at first I misunderstood this but I see what you mean now, thanks for the sharp eye